cmd/envtool/tests.go

```diff
@@ -112,13 +117,19 @@ func runGoTest(ctx context.Context, args []string, total int, times bool, logger
 
 		// logger.Desugar().Info("decoded event", zap.Any("event", event))
 
-		if event.Test != "" {
-			if results[event.Test] == nil {
-				results[event.Test] = &testResult{
-					outputs: make([]string, 0, 2),
-				}
+		// Skip processing if event.Test is empty
+		if event.Test == "" {
+			continue
+		}
```

Comment:
We should not do that because there are package-level events that we want to track. This change also broke tests. Please run them locally.